oer-bookr/src/containers/BookDescription.js

```diff
@@ -19,44 +21,163 @@ class BookDescription extends Component {
         {
           id: 0,
           reviewer: "Anonymous",
-          review:
+          text:
             "Students follow specific procedures to complete much of the work. Using this as an assessment may yield faulty information."
         },
         {
           id: 1,
           reviewer: "L.J",
-          review:
+          text:
             "Worksheets and support tell student the procedure to complete the work."
         }
-      ]
+      ],
+      modal: false,
+      reviewer: "",
+      text: "",
+      warning: false,
+      reviewId: ""
     }
   }
+
+  componentDidMount = () => {
+    // fetch the book information and the reviews array
+
+    const reviewer = localStorage.getItem("currentUser")
+    if (reviewer) {
+      this.setState({ reviewer: reviewer })
```

Comment:
When you're specifying the value of an object literal's key, you can say it just once if the variable with the value stored has the same name as the key. For example:
```javascript
const reviewer = localStorage.getItem("currentUser")
if (reviewer) {
  this.setState({ reviewer })
}
```